README.md

```diff
@@ -1,53 +1,16 @@
 
-## Feature Areas  
-- Development  
-  - Repo [GitHub & GitLabs first, future goal to go decentralized]  
-  - Automated builds and publishing steps.  
-- Project Management [tasks, priority]  
-  - Tasks  
-  - Priority  
-  - Reputation  
-  - Hook: communication, finances, legal  
-- Finance  
-  - CLOSED LOOP  
-  - Payroll  
-  - Funding  
-  - Dividends % Ownership  
-  - Revenue Stream  
-- Community  
-  - Personal Profiles [reputation]  
-  - Activity Feeds  
-  - Chat [adhoc, community, instant access]  
-  - Forums [cataloged discussion for specific thing] [comments on a task?]  
-- Legal  
-  - Catch all for things not facilitated on chain. Cannot call into question the on chain process’ and override their existing nature.  
-  - Documents  
-  - Courts  
-  - RL Jurisdiction  
+Please see our ["Key Scenarios"](./key-scenarios.md) document where we will document the user scenarios we will be developing solutions for.  
 
-## Scenarios  
-- See project on GitHub, click on button, buy some shares (if open to investment).  
-- Starting a new company (kickstarter ish).  
-- Established companies that want to create edge developments. Fund development.  
-- [Dividend] Verify the dividend stream (app to smart contract).  
-  - Verification (automated) to protect against fraud.  
-  - Company’s main address, easy.  
-  - They ship on our standard platform, easier for developer.  
-  - Closing the loop (stake holders, DAO, source code, income stream, back to dao & holders).  
-- Insight into your company’s different stats: revenue stream, investment distribution, development activity.  
-- Automated & decentralized environment, ENS pointing to it.  
-
-## Tech  
-- Project Management: Colony, DAOStack, Aragon  
-- Finance: OpenZeppelin  
-- Community: Status, UPort  
-- Legal: Aragon  
+## Contribute  
+Interested in this mission? Find something missing? We'd love for you to contribute! You can also make contact with us online, anything works. A link to our group chat is listed here: https://github.com/dOrgTech/.
 
 ## dOrg?  
-The term DAO is often only associated with "The DAO" hack. We also don't want to be vague. By simply saying "Decentralized Organization" it is self discriptive and simple to understand. Shortening those two words gives you "dOrg".  
+We wanted something that was easy for anyone, inside or outside the blockchain community, to remember and become familiar with. We think that the term dApp was extremely successful in doing this for Decentralized Apps, which brought us to thinking dOrg for Decentralized Organizations was a good fit.  
+
+The term DAO has become a standard in the blockchain community, and we do not aim to replace. We plan to use the term DAO in all technical documentation, and will only use dOrg as an up front "new user friendly" term.  
```

Comment:
Ahhhh okay, I definitely think we should form consensus on this soon. The term dOrg is funny, and a bit catchy, and I think we should figure out just how we should be using along side the terminology that already exists (DAO).

Reply:
I like dOrg it's fitting and it has legs.

Reply:
I think it will be easier just to call a DAO a DAO, and that using the term dOg exchangeable with a DAO will create confusion. Especially since DAO is sort of an accepted therm already.

But if both of you want to use DAO and dOrg exchangeably then, of course, I accept that.

Reply:
I think maybe we can call DAOs we've created through the Dapp "dOrgs", and all others are just vanilla DAOs?

Reply:
I'd be game to have a chat about this, as my mind isn't fully made up yet based on this discussion here. It seems like it could go either way.